pr-analysis/README.md

```diff
@@ -0,0 +1,32 @@
+# PR Analysis on operator-test-playbooks
+
+## How to run the code:
+
+### Install Necessary Modules
+`pip install -r requirements.txt`
+
```

Comment:
Suggest another line here like: 

`Note: you may need to install the python3-tkinter package for graphs to display.`

Reply:
done!